monkey/infection_monkey/system_info/system_info_collectors_handler.py

```diff
@@ -0,0 +1,35 @@
+import logging
+from typing import Sequence
+
+from infection_monkey.system_info.system_info_collector import SystemInfoCollector
+from telemetry.system_info_telem import SystemInfoTelem
```

Comment:
how does it work? don't you need to start with `infection_monkey`?

Reply:
26355540